firebase/deploy

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+main() {
+    set -euo pipefail
+    set -x
+
+    if (( $# > 1 )); then
+        exit_error "usage [CHANNEL=dev]"
+    fi
+
+    on_ci && setup_ci_creds
+
+    channel="${1:-dev}"
+    if [[ "${channel}" == "live" ]]; then
+        firebase --config firebase/firebase.json deploy --only hosting
+        exit 0
+    fi
+
+    pr_num=${GITHUB_REF:-unknown-pr}
+    pr_num=${GITHUB_REF#refs/pull/} #  GITHUB_REF == "refs/pull/:PR_NUM/merge"
```

Comment:
oh sorry, didn't notice this until now. this `GITHUB_REF` needs to be `pr_num`

Reply:
oups!